pkg/sql/pgwire/types.go

```diff
@@ -129,7 +129,11 @@ func (b *writeBuffer) writeTextDatum(
 		b.write([]byte(result))
 
 	case *tree.DUuid:
-		b.writeLengthPrefixedString(v.UUID.String())
+		// Start at offset 4 because `putInt32` clobbers the first 4 bytes.
+		s := b.putbuf[4 : 4+36]
```

Comment:
what guarantees that putbuf will always be large enough for this?

Reply:
It's a constant 64 bytes: `putbuf [64]byte`